.circleci/config.yml

```diff
@@ -1,4 +1,55 @@
-default_env_vars: &default_env_vars
+---
```

Comment:
This file is auto generated via `generate.sh` but don't see `config-2.2.yml` modified (or the patch file).  Can you make sure to make the changes in `config-2.2.yml`?

https://github.com/apache/cassandra/blob/trunk/.circleci/readme.md

Here is what I did last time

```
cd .circleci
BASEDIR=$PWD

# If changing CCM settings or modifying the executors, perform the following
# apply the patch before modifying config-2_1.yml if you are modifying the top part of the file; if touching after the executors are defined then can modify just the config-2_1.ym file and ignore the step of applying the patch and diff
patch -o $BASEDIR/config-2_1.yml.HIGHRES $BASEDIR/config-2_1.yml $BASEDIR/config-2_1.yml.high_res.patch
diff config-2_1.yml config-2_1.yml.HIGHRES > config-2_1.yml.high_res.patch

# make the changes required then generate and update defaults
./generate.sh
cp config.yml.LOWRES config.yml
```